### PR TITLE
fix risk of the list projection for unrelated piece types being alter…

### PR DESCRIPTION
**Pull request by boutell**
…ed due to copy by reference

See DC 1016

### CODE REVIEW

**Comment by boutell** [comment]
cc @jogarijo

File: lib/modules/apostrophe-pieces/lib/api.js [hunk 1]
```diff
@@ -616,6 +616,8 @@ module.exports = function(self, options) {
         // add the new field to the columns
         self.columns.push(newColumn);
       });
+      // So we do not inadvertently modify it for other subclasses
```
Comment (falkodev) [inline lib/modules/apostrophe-pieces/lib/api.js:619]: Is it to be sure `listProjection` will always be an object?

**Review by abea — APPROVED** [review]

File: lib/modules/apostrophe-pieces/lib/api.js (hunk 1 above)
Comment (boutell) [reply lib/modules/apostrophe-pieces/lib/api.js:619]: Yes there is a default setting for it.

**Comment by boutell** [comment]
@falkodev yes it is always an object, there is a default setting for it.

**Review by falkodev — APPROVED** [review]

**Pull request merged**